<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tableWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="fcfs">
         <attribute name="title">
          <string>FCFS</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="5" column="0">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QTableWidget" name="tableWidgetfcfs">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;table&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Arrival time</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Burst time</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Enter nomber of process</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Average waiting time</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLineEdit" name="lineEditfcfs"/>
          </item>
          <item row="5" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_10">
              <property name="text">
               <string>submit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="push_Buttonclear">
              <property name="text">
               <string>clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="sif">
         <attribute name="title">
          <string>SJF</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEditsjf"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QTableWidget" name="tableWidgetsjf">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;table&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Arrival time</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Burst time</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>submit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="push_Buttonclear1">
              <property name="text">
               <string>clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1" rowspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QRadioButton" name="radioButton">
              <property name="text">
               <string>Premetive</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_2">
              <property name="text">
               <string>Non-Perimitive</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLineEdit" name="lineEdit_4"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Enter nomber of process</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Average waiting time</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="roundroubin">
         <attribute name="title">
          <string>Round Robin</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Average waiting time</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLineEdit" name="lineEdit_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>30</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QTableWidget" name="tableWidgetrr">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;table&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Arrival time</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Burst time</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>submit</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="push_Buttonclear2">
              <property name="text">
               <string>clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Enter nomber of process</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Quatum</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_7"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLineEdit" name="lineEditrr"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="priority">
         <attribute name="title">
          <string>Priority</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEditper"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Enter nomber of process</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QRadioButton" name="radioButton_3">
              <property name="text">
               <string>Premetive</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_4">
              <property name="text">
               <string>Non-Perimitive</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QTableWidget" name="tableWidgetper">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;table&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Arrival time</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Burst time</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Piority</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Average waiting time</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>submit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="push_Buttonclear3">
              <property name="text">
               <string>clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLineEdit" name="lineEdit_9"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
